# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/certbot/no-download.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- certbot-4.1.1/certbot/setup.py.vanilla	2025-06-28 19:14:28.855922205 +0200
+++ certbot-4.1.1/certbot/setup.py	2025-06-28 19:14:46.139384851 +0200
@@ -25,22 +25,6 @@
 # here to avoid masking the more specific request requirements in acme. See
 # https://github.com/pypa/pip/issues/988 for more info.
 install_requires = [
-    # We specify the minimum acme version as the current Certbot version for
-    # simplicity. See https://github.com/certbot/certbot/issues/8761 for more
-    # info.
-    f'acme>={version}',
-    'ConfigArgParse>=1.5.3',
-    'configobj>=5.0.6',
-    'cryptography>=43.0.0',
-    'distro>=1.0.1',
-    'importlib_metadata>=8.6.1; python_version < "3.10"',
-    'josepy>=2.0.0',
-    'parsedatetime>=2.4',
-    'pyrfc3339',
-    'pytz>=2019.3',
-    # This dependency needs to be added using environment markers to avoid its
-    # installation on Linux.
-    'pywin32>=300 ; sys_platform == "win32"',
 ]
 
 dev_extras = [
